template/base/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  testMatch: ['<rootDir>/**/__tests__/*(*.)@(spec|test).[jt]s?(x)'],
+  moduleDirectories: ['node_modules'],
+  testPathIgnorePatterns: ['<rootDir>/dist', '<rootDir>/out/']
```

Comment:
We must improve this config to take care of many other things. 

Coverage, Mocking etc